src/activities/AddGeoreferenceToPdf.ts

```diff
@@ -0,0 +1,175 @@
+import type { IActivityHandler } from "@vertigis/workflow";
+import {
+    PDFArray,
+    PDFDict,
+    PDFDocument,
+    PDFName,
+    PDFNumber,
+    PDFString,
+} from "pdf-lib";
+
+export interface AddGeoreferenceToPdfInputs {
+    /**
+     * @description The source PDF document to add the georeference metadata to. Either a file path or byte content are accepted.
+     * @required
+     */
+    source: Blob;
+
+    /**
+     * @description The bounding coordinates of the map element to georeference in PDF user space.
+     * The array must contain two coordinate pairs in the following order: [[bottom-left x, bottom-left y], [top-right x, top-right y]].
+     * The coordinate [0, 0] represents the bottom left corner of the page.
+     * Typically the units of a PDF are in points (1/72 of an inch).
+     * @required
+     */
+    pageBounds: number[][];
+
+    /**
+     * @description The bounding coordinates of the map element to georeference in geographic space.
+     * The array must contain four coordinate pairs in the following order: [[bottom-left latitude, bottom-left longitude], [top-left latitude, top-left longitude], [top-right latitude, top-right longitude], [bottom-right latitude, bottom-right longitude]].
+     * @required
+     */
+    mapBounds: number[][];
+
+    /**
+     * @description The Well-Known Text (WKT) definition of the coordinate system of the map coordinates.
+     * @required
+     */
+    coordinateSystem: string;
```

Comment:
Would `coOrdinateSystem` be too pedantic here? I know some capitalizations don't always follow the general rule.

Either way, though, I still would recommend adding:
```
@displayName Co-ordinate System
```

Reply:
Sure, fair enough. I had still wondered about `coOrdinateSystem` but it's not important.